### ISSUE TITLE
mapState is not a function

### ISSUE THREAD

**Issue by KristerV** [saikat/react-apollo-starter-kit#15]
To reproduce just start the server and click the bottom link going to /http://localhost:3000/local-counter

[Why I'm using port 3000](https://github.com/saikat/react-apollo-starter-kit/issues/16)

```
17:33:33 server.1  |  backend error TypeError: mapState is not a function
17:33:33 server.1  |      at Connect.configureFinalMapState (/home/krister/code/markdownsites-apollo/node_modules/react-redux/lib/components/connect.js:154:
17:33:33 server.1  |  >  27)
17:33:33 server.1  |      at Connect.computeStateProps (/home/krister/code/markdownsites-apollo/node_modules/react-redux/lib/components/connect.js:141:23)
17:33:33 server.1  |      at Connect.updateStatePropsIfNeeded (/home/krister/code/markdownsites-apollo/node_modules/react-redux/lib/components/connect.js:20
17:33:33 server.1  |  >  3:35)
17:33:33 server.1  |      at Connect.render (/home/krister/code/markdownsites-apollo/node_modules/react-redux/lib/components/connect.js:338:40)
17:33:33 server.1  |      at /home/krister/code/markdownsites-apollo/node_modules/react/lib/ReactCompositeComponent.js:793:21
17:33:33 server.1  |      at measureLifeCyclePerf (/home/krister/code/markdownsites-apollo/node_modules/react/lib/ReactCompositeComponent.js:74:12)
17:33:33 server.1  |      at ReactCompositeComponentWrapper._renderValidatedComponentWithoutOwnerOrContext (/home/krister/code/markdownsites-apollo/node_mod
17:33:33 server.1  |  >  ules/react/lib/ReactCompositeComponent.js:792:27)
17:33:33 server.1  |      at ReactCompositeComponentWrapper._renderValidatedComponent (/home/krister/code/markdownsites-apollo/node_modules/react/lib/ReactC
17:33:33 server.1  |  >  ompositeComponent.js:819:34)
17:33:33 server.1  |      at ReactCompositeComponentWrapper.performInitialMount (/home/krister/code/markdownsites-apollo/node_modules/react/lib/ReactComposi
17:33:33 server.1  |  >  teComponent.js:361:30)
17:33:33 server.1  |      at ReactCompositeComponentWrapper.mountComponent (/home/krister/code/markdownsites-apollo/node_modules/react/lib/ReactCompositeCom
17:33:33 server.1  |  >  ponent.js:257:21)
17:33:33 server.1  |  backend error TypeError: mapState is not a function
17:33:33 server.1  |      at Connect.configureFinalMapState (/home/krister/code/markdownsites-apollo/node_modules/react-redux/lib/components/connect.js:154:
17:33:33 server.1  |  >  27)
17:33:33 server.1  |      at Connect.computeStateProps (/home/krister/code/markdownsites-apollo/node_modules/react-redux/lib/components/connect.js:141:23)
17:33:33 server.1  |      at Connect.updateStatePropsIfNeeded (/home/krister/code/markdownsites-apollo/node_modules/react-redux/lib/components/connect.js:20
17:33:33 server.1  |  >  3:35)
17:33:33 server.1  |      at Connect.render (/home/krister/code/markdownsites-apollo/node_modules/react-redux/lib/components/connect.js:338:40)
17:33:33 server.1  |      at /home/krister/code/markdownsites-apollo/node_modules/react/lib/ReactCompositeComponent.js:793:21
17:33:33 server.1  |      at measureLifeCyclePerf (/home/krister/code/markdownsites-apollo/node_modules/react/lib/ReactCompositeComponent.js:74:12)
17:33:33 server.1  |      at ReactCompositeComponentWrapper._renderValidatedComponentWithoutOwnerOrContext (/home/krister/code/markdownsites-apollo/node_mod
17:33:33 server.1  |  >  ules/react/lib/ReactCompositeComponent.js:792:27)
17:33:33 server.1  |      at ReactCompositeComponentWrapper._renderValidatedComponent (/home/krister/code/markdownsites-apollo/node_modules/react/lib/ReactC
17:33:33 server.1  |  >  ompositeComponent.js:819:34)
17:33:33 server.1  |      at ReactCompositeComponentWrapper.performInitialMount (/home/krister/code/markdownsites-apollo/node_modules/react/lib/ReactComposi
17:33:33 server.1  |  >  teComponent.js:361:30)
17:33:33 server.1  |      at ReactCompositeComponentWrapper.mountComponent (/home/krister/code/markdownsites-apollo/node_modules/react/lib/ReactCompositeCom
17:33:33 server.1  |  >  ponent.js:257:21)
17:33:33 server.1  |  [nodemon] app crashed - waiting for file changes before starting...
```

**Comment by saikat** [comment]
Believe this is fixed.  Duplicate of #12

**Comment by KristerV** [comment]
awesome!
